packages/dev/core/src/Shaders/ShadersInclude/pbrBlockReflectivity.fx

```diff
@@ -32,6 +33,11 @@ reflectivityOutParams reflectivityBlock(
 #ifdef METALLICWORKFLOW
     , in vec3 surfaceAlbedo
     , in vec4 metallicReflectanceFactors
+#endif
+    , in float baseDiffuseRoughness
+#ifdef BASE_DIFFUSE_ROUGHNESS
+    , in float baseDiffuseRoughnessTexture
+    , in vec2 vBaseDiffuseRoughnessInfos

```

Comment:
```suggestion
    , in vec2 baseDiffuseRoughnessInfos
```
to avoid potential mismatch with `vBaseDiffuseRoughnessInfos`, which is the uniform passed to the shader.